kolibri/plugins/learn/assets/src/views/HybridLearningContentCardListView.vue

```diff
@@ -1,86 +1,60 @@
 <template>
 
-  <div
-    class="card drop-shadow"
-    :class="[
-      { 'mobile-card': isMobile }
-    ]"
-  >
-    <router-link
-      :to="link"
+  <router-link :to="link" :class="$computedClass({ ':focus': $coreOutline })">
+    <div
```

Comment:
make `router-link` the _outer_ element

Reply:
Hrm - the point of having the div as the outer element was to make sure that the footer was not part of the clickable area of the card though, how is that handled now?

Reply:
There is no visual footer area, so users have no way of knowing what's clickable and what's not.

Instead, only interactive elements inside the card should "steal" clicks. This is done using `@click.prevent` like this:

https://github.com/learningequality/kolibri/pull/8944/files#r771528976

Reply:
OK, that's consistent with other places we've done this - would be good to revisit this in terms of what the UI should look like for users.